--- nxcomp/Loop.cpp	2010-03-30 15:38:11.000000000 +0200
+++ nxcomp.new/Loop.cpp	2010-07-11 15:21:00.487212748 +0200
@@ -10986,7 +10986,7 @@
                                       packMethod == PACK_LOSSLESS ||
                                           packMethod == PACK_ADAPTIVE)
   {
-    const char *dash = rindex(opt, '-');
+    const char *dash = (char *)rindex(opt, '-');
 
     if (dash != NULL && strlen(dash) == 2 &&
             *(dash + 1) >= '0' && *(dash + 1) <= '9')
